### ISSUE TITLE
[feature] Add aria-label attribute to slider handles

### ISSUE THREAD

**Issue by tinytinytinytiny** [simeydotme/svelte-range-slider-pips#93]
**Describe the feature**
I'd like to set the `aria-label` attributes on the range handles

**Explain it's value / reasoning**
Elements with `role="slider"` require an accessible name

**Additional context**
Add a `labels` property that accepts an array of a length equal to that of the one passed into `values`, i.e.

```
<RangeSlider values={[0, 50]} labels={["Starting Level", "Ending Level"]} />
```

**Comment by simeydotme** [comment]
I have no idea how the `label` was missed.

However, I need to dive deeper, on a surface check it seems that the label should be for the _whole_ slider, not individual thumbs. The thumbs already have `aria-valuetext` for that, but the whole slider does need a `aria-label` or `aria-labelledby` for sure!

Thanks for raising, and feel free to drop any more info/links if you have em :)

**Comment by tinytinytinytiny** [comment]
`aria-valuetext` is meant to be a human-readable text alternative of the `aria-valuenow` attribute when its value cannot be meaningfully represented by a number, i.e. (`aria-valuetext="8% (34 minutes) remaining"`). this is not meant to serve as a label for the control https://w3c.github.io/aria/#aria-valuetext

the ARIA APG provides an [example of a multi-thumb slider](https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-multithumb.html) that provides an `aria-label` for each moveable thumb

**Comment by simeydotme** [comment]
hi @tinytinytinytiny , so sorry this has been a long time! 🙏 

Can you take a look at this commit https://github.com/simeydotme/svelte-range-slider-pips/commit/6530136f64625df7d508b1eb9a40506560fa467a#diff-8361d0a16c946316e9bb97b31b167819e94934619153783444beca2a671cb499 and see if you think this is right?

I've tried to detect a `ariaLabels` array property, and if it's not provided; default it to say either;
- `Minimum value in range from ${min} to ${max}`
or
- `Maximum value in range from ${min} to ${max}`
or
- `Value ${i + 1} in range from ${min} to ${max}`;

depending on whether it's a range slider or just a thumb slider.

---

What do you think? any suggestions on the default text? (it needs to be super generic)
